src/utils/extraActionCreators/downloadProcessedMatrix.js

```diff
@@ -23,7 +22,6 @@ const downloadProcessedMatrix = (experimentId) => async (dispatch, getState) =>
 
     const body = {
       name: taskName,
```

Comment:
I assume that this is part of the bug you referred to earlier today -- how when downloading a Seurat object, the worker needs to receive the ETag for the embedding

Reply:
yes and no, the embedding is added in the API the. The UI will do nothing